python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -558,8 +563,7 @@ def joinMatchListWithCatalog(self, matchCat, sourceCat):
         """
         return joinMatchListWithCatalogImpl(self, matchCat, sourceCat)
 
-    @classmethod
-    def getMetadataBox(cls, bbox, wcs, filterName=None, photoCalib=None, epoch=None, bboxPadding=100):
+    def getMetadataBox(self, bbox, wcs, filterName=None, photoCalib=None, epoch=None):
```

Comment:
This needs to be changed back, see other comment

Reply:
See my comment above.

Reply:
Effectively reverted.